kafka/strimzi/templates/pool.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kafka.strimzi.io/v1beta2
```

Comment:
This will work as a first cut, but I've seen issues with scaling-up/down. The settings that I've ended up with are here: https://github.com/SeldonIO/gcp-infra-automation-prototype/blob/main/kafka/strimzi/templates/cluster.yaml

Reply:
We can leave those changes for a future PR

Reply:
I am copying across most of the changes.

Reply:
To clarify one change in particular, the split in two volumes, with one of type `type: ephemeral`, was because with KRaft the cluster gets an ID that gets persisted on that volume. 

On scale-down followed by scale-up, the cluster comes up with a new ID but the old volume is loaded showing the past ID. KRaft gets confused and stops the cluster from actually loading.

With an ephemeral volume, the persisted ID data goes away on scale-down and is recreated fresh on scale-up.

Reply:
added now in [6f9119c](https://github.com/SeldonIO/seldon-core/pull/6146/commits/6f9119c0065b84545287fea6e1d9c38db156a1c9)